app/Models/User.php

```diff
@@ -570,6 +570,38 @@ public function getRememberTokenName(): string
         throw new BadMethodCallException('Not implemented');
     }
 
+    /**
+     * Graduation Semester is a 6-digit code by default.
+     * To retrieve a more readable form, this method obtains the
+     * code for this User and translates it to the format "Fall xxxx.
+     */
+    public function getHumanReadableSemester(): string
```

Comment:
if you append `Attribute` to this function name, laravel will magically turn this into a User model attribute called `human_readable_semester`